packages/components/src/internal/components/labels/LabelsConfigurationPanel.tsx

```diff
@@ -0,0 +1,352 @@
+                rows: [templateToSave],
+            })
+                .then(() => {
+                    onActionCompleted(templateToSave.rowId);
+                    setSaving(false);
+                })
+                .catch(reason => {
+                    setError(resolveErrorMessage(reason, 'template', 'templates', 'updating'));
+                    setSaving(false);
+                });
+        } else {
+            insertRows({
+                schemaQuery: LABEL_TEMPLATE_SQ,
+                rows: List([templateToSave]),
+            })
+                .then(() => {
+                    onActionCompleted(templateToSave.rowId);
+                    setSaving(false);
+                })
+                .catch(response => {
+                    setError(resolveErrorMessage(response.get('error'), 'template', 'templates', 'inserting'));
+                    setSaving(false);
+                });
+        }
+    }, [onActionCompleted, updatedTemplate]);
+
+    return (
+        <>
+            {!updatedTemplate && template !== null && (
+                <div className="choices-detail__empty-message">Select template to view details.</div>
+            )}
+            {updatedTemplate && (
+                <form className="form-horizontal content-form choices-detail__form">
+                    {error && <Alert>{error}</Alert>}
+                    <FormGroup>
+                        <div className="col-sm-4">
+                            <DomainFieldLabel label="Name" required />
+                        </div>
+                        <div className="col-sm-8">
+                            <input
+                                className="form-control"
+                                name="name"
+                                onChange={onFormChange}
+                                disabled={saving}
+                                placeholder="Enter label's display name"
+                                type="text"
+                                value={updatedTemplate.name ?? ''}
+                            />
+                        </div>
+                    </FormGroup>
+                    <FormGroup>
+                        <div className="col-sm-4">
+                            <DomainFieldLabel label="Description" />
+                        </div>
+                        <div className="col-sm-8">
+                            <textarea
+                                className="form-control"
+                                name="description"
+                                onChange={onFormChange}
+                                disabled={saving}
+                                value={updatedTemplate.description ?? ''}
+                            />
+                        </div>
+                    </FormGroup>
+                    <FormGroup>
+                        <div className="col-sm-4">
+                            <DomainFieldLabel label="File Path" required />
```

Comment:
I will not this in the manual testing log, but when I go to an existing SM folder that went through the upgrade script to ensure the list exists, the name and path fields aren't require so I am able to save a new label template without those values. If the list gets created via the client side Domain.create, the fields are marked as required and the error message shows as expected when I click save without those values.

Reply:
fixed